docs/technical_reference/unit_models/pressure_exchanger.rst

```diff
@@ -0,0 +1,99 @@
+Pressure Exchanger
+==================
+This pressure exchanger unit model:
+    * is 0-dimensional
+    * is isothermal
+    * supports a single liquid phase only
+    * supports steady-state only
```

Comment:
We may want to mention other model assumptions:

- no mixing takes place between streams (in reality there is some mixing)
- ERD has no leakage
- both stream flow rates are equal, i.e., the ERD has "balanced flow" (which is ideal but not necessary)